packages/main/src/Table.ts

```diff
@@ -666,7 +657,7 @@ class Table extends UI5Element {
 	}
 
 	get _shouldRenderGrowing() {
-		return this.rows.length && this._growing?.hasGrowingComponent();
+		return !!this.rows.length && this._growing?.hasGrowingComponent();
 	}
 
 	get _growing() {
```

Comment:
I guess we had this calculated `_growing` property for the hbs. Since we have tsx now we can call `_getGrowing()` as method should we remove this calculated property and use the method version `_getGrowing()` like the other features `_getSelection()`, `_getVirtualizer()` ...